### DISCUSSION TITLE
I don't know how to use it

### DISCUSSION THREAD

**Discussion by kuki6** [StevenBlack/hosts#1979] [Q&A]
I want to ask a question.Why are all the addresses in my host file 0.0.0.0? There is no way to resolve without a specific IP address

**Comment by StevenBlack** [comment]
Hi @kuki6 thank you for your question.

This is explained in the readme: in the section titled [We recommend using 0\.0\.0\.0 instead of 127\.0\.0\.1](https://github.com/StevenBlack/hosts).

Note that if you generate your own hosts file using the code in this repository, you can specify the IP address to use.  This is documented in the [Command line options](https://github.com/StevenBlack/hosts) section in the readme.
